Test/SchemaCompilerSpec.hs

```diff
@@ -148,7 +148,7 @@ tests = do
             it "should compile CanUpdate instance with an array type with an explicit cast" do
                 let statement = StatementCreateTable $ CreateTable {
                     name = "users",
-                    columns = [ Column "id" PUUID Nothing False True Nothing, Column "ids" (PArray PUUID) Nothing False False Nothing],
+                    columns = [ Column "id" PUUID Nothing True True Nothing, Column "ids" (PArray PUUID) Nothing False False Nothing],
```

Comment:
I had to change a whole bunch of existing unit tests, that for some reason specified that the "id" column of their table is nullable.

To me this looked like a simple copy-paste error, but I'd love it if someone confirmed my suspicion.

Reply:
Yes 👍 looks like a copy paste error to me as well